### ISSUE TITLE
give notify popup some more appropriate ARIA attributes

### ISSUE THREAD

**Issue by moloko** [adaptlearning/adapt_framework#2052]
#2051 suggests adding `aria-haspopup="dialog"` to hot graphic pins, in which case the Notify popup will need to have a matching role of `"dialog"`.

In addition it has been suggested that, as a modal popup, it should have an `aria-modal` attribute

**Comment by moloko** [comment]
done in 130cd0fe08121c36d8feca0e765b108bbfd53af3
